straxen/plugins/led_calibration.py

```diff
@@ -0,0 +1,114 @@
+'''
+Dear nT analyser, 
+if you want to complain please contact: chiara@physik.uzh.ch, gvolta@physik.uzh.ch
+'''
+
+import strax
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+import numba
+from numba import njit
+
+#export, __all__ = strax.exporter()
+# QUESTION: what does 'export, __all__ = strax.exporter()' do?
+# ANSWER: [fill in]
+
+@strax.takes_config(
+    strax.Option('LED_window',
+                 default=(125, 250),
+                 help="Window (samples) where we expect the signal in LED calibration"),
+    strax.Option('noise_window',
+                 default=(350, 475),
+                 help="Window (samples) to analysis the noise"),
+    strax.Option('light_level',
```

Comment:
I feel like `light_level` isn't the best name for this config option. Maybe `channel_list`?

Reply:
Got it! I will change it